bin/reth/src/db/snapshots/mod.rs

```diff
@@ -214,6 +214,7 @@ impl Command {
         for snap in &snapshots {
             let start_time = Instant::now();
             let jar = NippyJar::<SegmentHeader>::load(snap.as_ref())?;
+            let _cursor = NippyJarCursor::new(&jar)?;
```

Comment:
do we need it?

Reply:
yes it's important, to make the benchmark closer to real usage